### PR TITLE
Add a way to configure connecting to local compositor instance in development and to production when deployed

### PR DESCRIPTION
**Pull request by wkazmierczak**
Default connection set to production, when you want to connect with dev use command `npm run dev`

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -3,17 +3,18 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "docusaurus": "docusaurus",
```
Comment (wkozyra95) [inline docs/package.json:6]: don't use NODE_ENV for that, do not modify other scrips, only `dev` script should be added, and old ones shoud stay as they were

File: docs/src/pages/playground.tsx [hunk 2]
```diff
@@ -97,6 +97,9 @@ function Homepage() {
     input_5: InputResolution.Resoultion1920x1080,
     input_6: InputResolution.Resoultion1920x1080,
   });
+  const { siteConfig } = useDocusaurusContext();
+  const environment = siteConfig.customFields.environment as string;
```
Comment (wkozyra95) [inline docs/src/pages/playground.tsx:101]: I don't like passing that through the components, this should be a global value. Is there a way to access that field without a hook?

Comment (wkazmierczak) [reply docs/src/pages/playground.tsx:101]: As far as I know and have tested, hook is needed, but I'll double-check.

File: docs/src/api.ts [hunk 3]
```diff
@@ -1,4 +1,7 @@
-const BACKEND_URL: URL = new URL('http://localhost:8081');
+// The @generated/docusaurus.config file is built at runtime, not present during linting.
+// Disabling the rule prevents linting errors for this valid import.
```
Comment (wkozyra95) [inline docs/src/api.ts:2]:
```suggestion
```

**Review by wkozyra95 — APPROVED** [review]

**Pull request merged**